api/libretimeapi/tests/test_views.py

```diff
@@ -1,12 +1,24 @@
+import datetime
 import os
 
 from django.conf import settings
 from django.contrib.auth.models import AnonymousUser
+from django.utils import dateparse
 from libretimeapi.views import FileViewSet
 from model_bakery import baker
 from rest_framework.test import APIRequestFactory, APITestCase
 
 
+def fromisoformat(datetime_string):
+    try:
+        datetime_obj = datetime.datetime.strptime(
+            datetime_string, "%Y-%m-%dT%H:%M:%S.%fZ"
+        )
+    except ValueError:
+        datetime_obj = datetime.datetime.strptime(datetime_string, "%Y-%m-%dT%H:%M:%SZ")
+    return datetime_obj.time()
```

Comment:
What moving this in a utils module and add more typing:
```py
from datetime import datetime

def fromisoformat(value: str) -> datetime:
    """
    Parse a iso formatted datetime string.
    """
    try:
        result = datetime.strptime(datetime_string, "%Y-%m-%dT%H:%M:%S.%fZ")
    except ValueError:
        result = datetime.strptime(datetime_string, "%Y-%m-%dT%H:%M:%SZ")
    return result.time()
```

Reply:
This is actually not needed and shouldn't be there. Will remove